llama-index-integrations/vector_stores/llama-index-vector-stores-azureaisearch/llama_index/vector_stores/azureaisearch/base.py

```diff
@@ -276,13 +279,26 @@ def _create_index(self, index_name: Optional[str]) -> None:
         metadata_index_fields = self._create_metadata_index_fields()
         fields.extend(metadata_index_fields)
         logger.info(f"Configuring {index_name} vector search")
+        # Determine the compression type
+        compressions = []
+        if self._compression_type == "binary":
```

Comment:
Should we enable oversampling/rescoring when binary compression is enabled, at least as a default?

Reply:
Rerank=True and Oversampling=4 are the defaults already.